File: .sdk/tm/php/utility/MakeContext.php
<?php
declare(strict_types=1);

// TiktokEngagementBot SDK utility: make_context

require_once __DIR__ . '/../core/Context.php';

class TiktokEngagementBotMakeContext
{
    public static function call(array $ctxmap, ?TiktokEngagementBotContext $basectx): TiktokEngagementBotContext
    {
        return new TiktokEngagementBotContext($ctxmap, $basectx);
    }
}

File: rb/utility/make_context.rb
# TiktokEngagementBot SDK utility: make_context
require_relative '../core/context'
module TiktokEngagementBotUtilities
  MakeContext = ->(ctxmap, basectx) {
    TiktokEngagementBotContext.new(ctxmap, basectx)
  }
end
